src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -316,8 +315,9 @@ def prepare_or_run_pipeline(
                     )
 
             # Construct S3 outputs from container for step
+            only_processing_step_possible = False
             outputs = None
-
+            output_path = None
```

Comment:
Seems like this is not used?

Reply:
Used in `sagemaker.estimator.Estimator` constructor.